scripts/tasks/statistics

```diff
@@ -18,11 +18,11 @@ fi
 cd "${FOGLAMP_ROOT}/python"
 
 os_name=`(grep -o '^NAME=.*' /etc/os-release | cut -f2 -d\" | sed 's/"//g')`
-os_version=`(grep -o '^VERSION_ID=.*' /etc/os-release | cut -f2 -d\" | sed 's/"//g')`
 
-if [[ ( $os_name == *"Red Hat"* || $os_name == *"CentOS"* ) &&  $os_version == *"7"* ]]; then
+if [[ $os_name == *"Raspbian"*  ]]; then
```

Comment:
With this, we can remove `cpulimit` from control files for aarch64 and x86_64 https://github.com/foglamp/foglamp-pkg/tree/develop/packages/Debian. 

Also I'm not sure, we should replace the check based on arch=`uname -m` Or arch=`arch`?!